Parameter 'aChannel' uses Hungarian Notation, consider using 'channel' instead  (mozilla/no-aArgs)

```sh
$ npm run eslint

> esper-pioneer-shield-study@1.0.4 eslint /Users/pdehaan/dev/github/mozilla/esper-pioneer-shield-study
> eslint . --ext=jsm,js,json

/Users/pdehaan/dev/github/mozilla/esper-pioneer-shield-study/addon/lib/StudyTelemetryCollector.jsm
  218:38  warning  Parameter 'aChannel' uses Hungarian Notation, consider using 'channel' instead  mozilla/no-aArgs
```

Looks like the warning is coming from https://github.com/motin/esper-pioneer-shield-study/blob/552d2c7f6aef2a31c426e1c7e8082215936bca22/addon/lib/StudyTelemetryCollector.jsm#L218-L223

Looks like the easy fix is to add an `// eslint-disable-next-line` style comment to ignore that one rule for that one specific line (or, rename function arguments...):

```js
// eslint-disable-next-line mozilla/no-aArgs
function getParentProcessScalars(aChannel, aKeyed = false, aClear = false) {
  ...
}
```

--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -7,17 +7,8 @@
  */
 
 module.exports = {
-  parserOptions: {
-    ecmaVersion: 8,
-    sourceType: "module",
-    ecmaFeatures: {
-      jsx: false,
-      experimentalObjectRestSpread: true,
-    },
-  },
   env: {
     es6: true,
-    // 'browser-window': false
   },
   extends: [
     "eslint:recommended",
@@ -26,16 +17,23 @@ module.exports = {
      */
     "plugin:mozilla/recommended",
   ],
-
+  parserOptions: {
+    ecmaVersion: 8,
+    sourceType: "module",
+    ecmaFeatures: {
+      jsx: false,
+      experimentalObjectRestSpread: true,
+    },
+  },
   plugins: ["json", "mozilla"],
-
+  root: true,
   rules: {
     "babel/new-cap": "off",
+    "mozilla/balanced-listeners": "off",
+    "mozilla/no-aArgs": "warn",
     "comma-dangle": ["error", "always-multiline"],
     eqeqeq: "error",
     indent: ["warn", 2, { SwitchCase: 1 }],
-    "mozilla/no-aArgs": "warn",
-    "mozilla/balanced-listeners": 0,
     "no-console": "warn",
     "no-shadow": ["error"],
     "no-unused-vars": "error",

--- a/addon/Config.jsm
+++ b/addon/Config.jsm
@@ -1,9 +1,8 @@
 "use strict";
 
-/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "(Config|EXPORTED_SYMBOLS)" }]*/
-
 const EXPORTED_SYMBOLS = ["Config"];
 
+// eslint-disable-next-line no-unused-vars
 const Config = {
   studyName: "esper-pioneer-shield-study",
   branches: [{ name: "pioneer", weight: 1 }],

--- a/addon/bootstrap.js
+++ b/addon/bootstrap.js
@@ -1,7 +1,6 @@
-"use strict" /* eslint no-unused-vars: ["error", { "varsIgnorePattern": "(startup|shutdown|install|uninstall)" }]*/; // Cu.import
+"use strict";
 
-/* global  __SCRIPT_URI_SPEC__  */
-/* global Feature, Services */ const { utils: Cu } = Components;
+const { utils: Cu } = Components;
 Cu.import("resource://gre/modules/Console.jsm");
 Cu.import("resource://gre/modules/Services.jsm");
 Cu.import("resource://gre/modules/XPCOMUtils.jsm");
@@ -51,6 +50,7 @@ this.Bootstrap = {
 
     const isEligible = await Pioneer.utils.isUserOptedIn();
     if (!isEligible) {
+      // eslint-disable-next-line no-console
       console.log(
         "Not eligable for Pioneer study. Will uninstall the study add-on.",
       );

--- a/addon/lib/Helpers.jsm
+++ b/addon/lib/Helpers.jsm
@@ -1,5 +1,3 @@
-/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "(EXPORTED_SYMBOLS|Helpers)" }]*/
-
 const EXPORTED_SYMBOLS = (this.EXPORTED_SYMBOLS = ["Helpers"]);
 
 this.Helpers = {

--- a/addon/lib/Pioneer.jsm
+++ b/addon/lib/Pioneer.jsm
@@ -1,5 +1,3 @@
-/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "(EXPORTED_SYMBOLS|Pioneer)" }]*/
-
 const { utils: Cu } = Components;
 Cu.import("resource://gre/modules/XPCOMUtils.jsm");
 

--- a/addon/lib/StudyTelemetryCollector.jsm
+++ b/addon/lib/StudyTelemetryCollector.jsm
@@ -1,6 +1,6 @@
 "use strict";
 
-/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "(EXPORTED_SYMBOLS|StudyTelemetryCollector)" }]*/
+/* eslint-disable no-console */
 
 const { utils: Cu } = Components;
 Cu.import("resource://gre/modules/Console.jsm");
@@ -47,6 +47,7 @@ XPCOMUtils.defineLazyModuleGetter(
   "chrome://esper-pioneer-shield-study/content/lib/Helpers.jsm",
 );
 
+// eslint-disable-next-line no-unused-vars
 class StudyTelemetryCollector {
   constructor() {}
 
@@ -215,6 +216,7 @@ class StudyTelemetryCollector {
 
     // firefox/browser/modules/test/browser/head.js
 
+    // eslint-disable-next-line mozilla/no-aArgs
     function getParentProcessScalars(aChannel, aKeyed = false, aClear = false) {
       const scalars = aKeyed
         ? Services.telemetry.snapshotKeyedScalars(aChannel, aClear).parent

